integration-tests/zombienet/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "zombienet-sdk-tests"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+subxt = { version = "0.35.0", default-features = false, features = ["native"] }
+tokio = "1.35.0"
+tracing-subscriber = "0.3.18"
+zombienet-sdk = { git = "https://github.com/paritytech/zombienet-sdk", tag = "v0.2.1" }
+log = "0.4"
+anyhow = "1.0.82"
+toml_datetime = "=0.6.5"
```

Comment:
is this dependency used?

Reply:
No, I pinned to workaround on a transitive dependency issue but is already fixed.
Thanks!!